splash/tests/test_render.py

```diff
@@ -368,15 +368,17 @@ def assertResponse200Get(self, code):
 
     @https_only
     def test_http2_supported(self):
-        r = self.request({'url': self.ts.mockserver.https_url("http-version")})
+        r = self.request({
+            'url': self.ts.mockserver.https_url("http-version"),
+            'http2': '1',
+        })
         self.assertStatusCode(r, 200)
         self.assertIn("http2", r.text)
 
     @https_only
-    def test_disable_http2_support(self):
+    def test_http2_disabled_by_default(self):
         r = self.request({
             'url': self.ts.mockserver.https_url("http-version"),
-            'http2': '0',
```

Comment:
I haven't checked it, but why isn't explicit disabling working?

Reply:
It does work, but since we had no test explicitly enabling, I did not though of having one explicitly disabling. I’ll add a test relying on the default but keep a test explicitly disabling.